sympy/calculus/accumulationbounds.py

```diff
@@ -698,6 +698,9 @@ def union(self, other):
         if other.min <= self.min and other.max >= self.min:
             return AccumBounds(other.min, Max(self.max, other.max))
 
+    @property
+    def func(self):
+        return self.__class__
```

Comment:
I think we've agreed that ultimately it's best for this class not to be an Atom although maybe making that change caused some problems somewhere. It's fine if we don't want to change that now but if this method is a workaround for that problem then it would be good to indicate that with a comment like:
```python
# XXX: This method is only needed because AccumBounds is a subclass
# of Atom even though it is not really an Atom. If AccumBounds was fixed
# to not subclass Atom then this func method could be deleted,
```

Reply:
Same goes for the other classes that have this

Reply:
I believe some of these classes have recently been changed. I think I should merge with master to make sure that there are no issues here.

Reply:
I got confused here due to other classes with the same name
```
sympy/physics/vector/vector.py:class Vector(Printable, EvalfMixin):
sympy/vector/vector.py:class Vector(BasisDependent):
sympy/physics/vector/dyadic.py:class Dyadic(Printable, EvalfMixin):
sympy/vector/dyadic.py:class Dyadic(BasisDependent):
```
Will add this comment